lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -79,6 +80,16 @@
     public static final int PUBLIC_ACCESS_GROUP = -1;
     public static final int ADMIN_ACCESS_GROUP = -2;
 
+    /** ElasticSearch max length of an HTTP URL defaults to 4kb.
```

Comment:
So this line still seems like it applies to the old field that was the url size rather than this new field that's the max # of objects. I'd expect something like 

"The maximum number of types that can be specified when restricting the search to specific types"

or something like that...

Reply:
Done.